### DISCUSSION TITLE
OSPOS Web Proposal/Demo

### DISCUSSION THREAD

**Discussion by WebShells** [opensourcepos/opensourcepos#3660] [Team Posts]
Hello Guys, started a new discussion to be noticed by members... 
Been discussing me and @jekkos about the need for a new website for our project where un-experienced users might be easier to access. 

The web sections will be as follows: 

- **Main Page:** including summary of the project 
- **About Us:** Some info about the organization, when it was started, members, etc... 
- **Features Section:** It includes a list of basic features and extended ( newly added ) features, can have announcements as well.
- **Demo Button:** Where user can check our demo site easily.
- **Contact us:** Regular contact us section
- **KnowledgeBase:** FAQ and ticketing support system divided into several categories, like:
Feature request, Bug report, Custom Development Request, etc...

For sure the main reason is to have more control over our web content and its display, offer more click to action buttons in order to offer the users more user friendly web interface to navigate through download, check code, demo, and alot of helpful info. 

@jekkos at first suggested to use jekyll linked to github where content will be loaded automatically, yet a basic way. I prefer to use more dynamic, neat, and classy web. Looking more modern to end user and gives our nowadays web experience.

So far with what we've discussed and got ready, much of missing content but as a basic layout, check the below:

[OSPOS Organization Web](http://ospos.wshells.org) 

This is still under discussion and no serious step was taken, except web layout and some features.
Suggestions are very welcomed, and **We'd like to know if jekyll is more preferred by you** 

Thanks for reading mates... 
@opensourcepos/org

**Comment by WebShells** [comment]
SEO & Responsiveness to be considered as well.

**Comment by odiea** [comment]
It looks good. If you think it will help users please continue with your fine work.

**Comment by WebShells** [comment]
@odiea am sorry how your character/avatar looks like ;p but am not sure how everyone looks in order to create his own character... 
Please for any suggestions don't hesitate to list it here...

**Comment by odiea** [comment]
That's my tartar @daN4cat suggested I use. I am Irish.

**Comment by WebShells** [comment]
So i should replace yours by @daN4cat avatars ;x  ? @daN4cat what about yours :x ?

**Comment by odiea** [comment]
No That is my avatar.

**Comment by WebShells** [comment]
Hahahaha okkkk great ;p nice to meet you man :D

**Comment by SteveIreland** [comment]
Agreed. However, I lay claim to the bald guy icon but I'll need to grow a beard again.

**Comment by objecttothis** [comment]
I think the reason that @jekkos was wanting to use Jekyll was to reduce the ongoing maintence need by a CMS or other such site.  I personally think that your demo looks great, but would suggest making sure that the project is as low maintenance as possible.  It also might be helpful to have links to bintray for people wanting to beta test.

**Comment by WebShells** [comment]
@objecttothis i agree from the maintenance part, already settled it up by listing the most important news/updates applied to the project ignoring minor ones. For the cms part am doing my best to reduce usage of unwanted plugins or trouble makers as said. Not worried for ongoing hosting, since I own this dedicated since a long period of time...
We might need to setup a list of most FAQ in order to make it  clear for user. 
Bintray links to be added, but with what i've done so far is a sample with missing content just to show layout and to deliver the idea.
Choice is yours Guys... Still need @daN4cat approval ...

**Comment by objecttothis** [comment]
My only other thoughts are to include the current OSPOS logo and tie together the styling of the app and the site. Personally I really like how clean the site looks and I don't dislike the GUI styling of OSPOS, but the two don't at all look similar. Food for thought.
________________________________

**Comment by WebShells** [comment]
@objecttothis i didn't apply the logo yet, the site title is just a formatted text... You mean we set the current OSPOS retail Bag ? With the circle inside ? The colors of the web were set to fit ospos flatly theme, dark and light blue as you can see all sections are titles with these colors or highlighted once hover... Suggestions are more than welcome in order to build a successful results ! (Y)

**Comment by objecttothis** [comment]
Yes I mainly meant the logo you referenced. Sorry, I was not aware of what was finished and what was just blocked out.
________________________________

**Comment by WebShells** [comment]
@objecttothis regarding your avatar, how do you look like ;p ? So i can apply the right outer shape :x 

@WebShells Works for me. I have chiseled abs and stunning features... just kidding. I'm a slightly overweight former powerlifter.  That should give you an idea of my body type.  Red hair (but not ginger). As for facial hair I usually keep a thin and short goatee or chin patch.  My interest in the OSPOS project is a means to an end.  We own a Christian bookstore and use OSPOS, so the additions, etc that I contribute are focused on what helps the community and helps the bookstore.  I give you that to see if that helps with my title. I don't have strong feelings either way.

**Comment by SteveIreland** [comment]
Since you're bald then you can have the bald guy.  The beard wasn't gray anyway.  :-)

**Comment by SteveIreland** [comment]
Also, you might want to link to the Wiki instead of the home page so that everyone knows that there is a WIki right from the start.

**Comment by WebShells** [comment]
@SteveIreland i can do another bald guy with a grey beard with dark skin and glasses :p... Anyway for the wiki thing, check the FAQ section i already listed main things.
Try to subnit a ticket and let me know if there is any issue in the process.

**Comment by odiea** [comment]
Well when I was in Jamaica about 20 years ago everyone called me Kenny Rodgers. So if you have something like that. LOL

**Comment by WebShells** [comment]
Will look that up 😂

**Comment by WebShells** [comment]
Updates On Features Section... Please check if you like the display or we should display it in another way @jekkos @daN4cat @SteveIreland @odiea @objecttothis 
[Features List](http://ospos.wshells.org/features/)

**Comment by odiea** [comment]
Looks good. Thanks for the work you are doing.

**Comment by jekkos** [comment]
@WebShells good work. Like the features overview. Maybe we can gather some comments form github for the recommendations section? Also should ask the users whether they agree on publishing their recommendatoin.
Where does the contact form go? Maybe we should just link to github for creating issues instead of sending emails for this?

**Comment by jekkos** [comment]
maybe we could consider making it a onepager? like adding features to the frontpage? might give a better overview

**Comment by WebShells** [comment]
Thanks @jekkos & @odiea for your  feedback... @jekkos as discussed we can add a direct link to github issues list and to create a new issue... For listing the features on main page i found it somehow copying what others are doing, phppos.com as they list the features on the main page. Thats why i preferred putting it under its own section..
With all of us, missing @daN4cat seems positive for it, ill proceed..
Ill post for public in order to get suggestions on what people would like to see on our web. 
Goooood day everybody!

**Comment by daN4cat** [comment]
I wear glasses, I have a beard, a bit more white hairs (that started early in my life when it was not so fashionable like nowadays).

**Comment by daN4cat** [comment]
Outside that it looks good. Well done for the good work.

**Comment by WebShells** [comment]
@daN4cat thanks for your feedback... Will try to update your avatar as mentioned (Y)... If you have any comments, suggestions, or sections that you'd like to be mentioned/listed on the website please let me know...

**Comment by WebShells** [comment]
Hello again @opensourcepos/org, 

With what all were worried about "Directing users away from git project by creating our own website", I've added a new section to the web that includes Live GitHub updates on the project.
I replaced the main homepage "We're active on git" by "We're Synced with Git" and after clicking it redirects users to Git Updates where we have real-time live synced issues, commits, and ospos releases hosted on Git. 
Added submitting an issue at the top of the page, and request a feature option for those who'd like to ask for a specific feature... Maybe will generate some donations and income to the organization. 
Still waiting for more suggestions to apply... The web will be ready the time you submit comments and i re-work. 
I re-arranged menu and listed contact us under Knowledge Base to lessen the load and make people search before submitting us a message.
Suggestions are more than welcomed again and again, please guys feel free to mention what best helps us.
Regards

**Comment by daN4cat** [comment]
I'll try to spend a bit of time going through the pages, I had just a skim through.
By the way, you can just put my name Francesco without the UK (although I live in London, UK). Thanks.

**Comment by WebShells** [comment]
Thanks for your time (Y)

**Comment by WebShells** [comment]
@opensourcepos/org was discussing with @jekkos regarding proceeding with the new web... Shall we ? or prefer staying on the default landing page ?
It's somehow a pending project that since i have sometime to spend at home after surgery, i can proceed with...
Thoughts ?
